internal/pkg/pluginmanager/manager.go

```diff
@@ -20,15 +27,15 @@ func DownloadPlugins(conf *configloader.Config) error {
 		pluginFileName := configloader.GetPluginFileName(&tool)
 		if _, err := os.Stat(filepath.Join(pluginsDir, pluginFileName)); errors.Is(err, os.ErrNotExist) {
 			// plugin does not exist
-			log.Printf("=== now downloading plugin: %s, version: %s ===", pluginFileName, tool.Version)
+			log.Printf("=== downloading plugin: %s ===", pluginFileName)
 			err := dc.download(pluginsDir, pluginFileName, tool.Version)
 			if err != nil {
 				return err
 			}
-			log.Printf("=== plugin: %s, version: %s downloaded ===", pluginFileName, tool.Version)
+			log.Printf("=== plugin: %s has been downloaded ===", pluginFileName)
 			continue
 		}
-		log.Printf("=== plugin: %s, version: %s exists ===", pluginFileName, tool.Version)
+		log.Printf("=== plugin: %s exists ===", pluginFileName)
```

Comment:
Let's take this chance to make it more user-friendly. I suggest:

"plugin: %s already exists in local plugin-dir %s"

What do you think?

Reply:
I already printed the plugin-dir above. So it's no need to print the dir info here.